### PR TITLE
feat: init charter

### PR DESCRIPTION
**Pull request by evenstensberg**
This PR effectively adopts (and replaces) the current governance model of `webpack` with the new governance model, pending approval from the CPC (Cross Project Council) from the OpenJS Foundation.

This proposal was crafted together with the Cross Project Council and current maintainers of webpack.

### This PR:

- Adds the CHARTER.md file

cc @webpack/core-team cc @webpack/tsc

### CODE REVIEW

**Comment by jhnns** [comment]
@sokra What do you think? :)

File: CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  - `sass-loader`
+  - `file-loader`
+  - `style-loader`
+  - `webpack-hot-middleware`
+  - `webpack-bundle-analyzer`
+
+These enable the handling of different asset types, interacting with webpack in various capacities to monitor builds 
+and improve applications.
+
+These projects, along with others housed within the `github.com/webpack` and `github.com/webpack-contrib`.
```
Comment (ovflowd) [inline CHARTER.md]: @evenstensberg I think we forgot the Out-of-scope section? Was it removed?

File: CHARTER.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
```
Comment (tobie) [inline CHARTER.md]: This sentence is unclear.

File: CHARTER.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+- **webpack-cli**: Offers a command-line interface to interact with webpack's build processes and configurations.
+- **webpack-contrib projects**: Includes various loaders, plugins, middlewares and utility repositories such as:
+  - `sass-loader`
+  - `file-loader`
+  - `style-loader`
+  - `webpack-hot-middleware`
+  - `webpack-bundle-analyzer`
+
+These enable the handling of different asset types, interacting with webpack in various capacities to monitor builds 
+and improve applications.
```
Comment (tobie) [inline CHARTER.md]: If the list isn't comprehensive, maybe shorten it, e.g.:
```suggestion
In addition to the main repository, `webpack` (bundler), the organization maintains and manages several other core 
projects that are integral to the ecosystem such as `webpack-dev-server`, `webpack-cli`,  and `webpack-contrib` projects.
```

File: CHARTER.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+These projects, along with others housed within the `github.com/webpack` and `github.com/webpack-contrib`.
+
+### 1.2: Out-of-Scope
+
+- Testing frameworks (e.g., unit, integration, or end-to-end testing tools)
+- API development and handling
```
Comment (tobie) [inline CHARTER.md]: Can you be a little more specific about API dev? is is web APIs?

File: CHARTER.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+- Server-side rendering directly (although it can integrate with SSR frameworks)
+- Frameworks and libraries that operate in their own capacity.
```
Comment (tobie) [inline CHARTER.md]:
```suggestion
- `webpack`-related frameworks and libraries that operate in their own capacity.
```

File: CHARTER.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+## Section 3. Establishment of the TSC
+
+TSC members can be either _regular_ members or _voting_ members. Regular members can attend meetings and participate in 
+TSC discussions, but do not vote. Voting members can do everything regular members can do, and also have the ability to 
+cast votes when consensus is not reached on an issue.
+
+TSC memberships are not time-limited. There is no maximum size of the TSC. The TSC must have at least four voting 
+members.
+
+There is no specific set of requirements or qualifications for TSC membership beyond these rules. The TSC may add
```
Comment (tobie) [inline CHARTER.md]: what does "these rules" mean here? Can you rephrase that sentence to make it more clear?

File: CHARTER.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+TSC members can be either _regular_ members or _voting_ members. Regular members can attend meetings and participate in 
+TSC discussions, but do not vote. Voting members can do everything regular members can do, and also have the ability to 
+cast votes when consensus is not reached on an issue.
+
+TSC memberships are not time-limited. There is no maximum size of the TSC. The TSC must have at least four voting 
+members.
+
+There is no specific set of requirements or qualifications for TSC membership beyond these rules. The TSC may add 
+additional voting members to the TSC by a standard TSC motion. A TSC member can be removed from the TSC by voluntary 
+resignation or by a standard TSC motion. A standard TSC motion can be used to change a regular TSC member to a voting
```
Comment (tobie) [inline CHARTER.md]: What is "a standard TSC motion"? Where is it defined?

File: CHARTER.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+voting membership shares an employer, then the situation must be immediately remedied by the removal of voting member 
+status from one or more TSC voting members affiliated with the over-represented employer(s).
+
+The TSC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On 
+Air, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson. 
+Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC voting member. 
+Minutes or an appropriate recording shall be taken and made available to the community through accessible public 
+postings.
+
+TSC voting members are expected to regularly participate in TSC activities.
```
Comment (tobie) [inline CHARTER.md]: What are TSC activities? Maybe define them more precisely.

File: CHARTER.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Project governance and process.
+* GitHub repository hosting.
+* Conduct guidelines.
+* Maintaining the list of additional Collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between Collaborators or Foundation projects.
```
Comment (tobie) [inline CHARTER.md]:
```suggestion
* Mediating technical conflicts between Collaborators or `webpack` projects.
```

File: CHARTER.md [hunk 10]
```diff
@@ -0,0 +1,192 @@
...
+## Section 4. Roles & Responsibilities of the TSC
+
+Subject to such policies as may be set by the CPC, the TSC voting members are responsible for all technical development 
+within the webpack project, including:
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Project governance and process.
+* GitHub repository hosting.
```
Comment (tobie) [inline CHARTER.md]: I'm not sure what "GitHub repository hosting" means here

File: CHARTER.md [hunk 11]
```diff
@@ -0,0 +1,192 @@
...
+The TSC voting members will define webpack project's release vehicles.
+
+### Section 4.1. webpack Project Operations
+
+The TSC voting members will establish and maintain a development process for the webpack project. The development 
+process will establish guidelines for how the developers and community will operate. It will, for example, establish 
+appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance 
+of a TSC meeting).
+
+The TSC and entire technical community will follow any processes as may be specified by the OpenJS Foundation Board
```
Comment (tobie) [inline CHARTER.md]: I'd frame this as more of a note. e.g.:

"Note that project operations remain subject to any relevant policy or process specified by the board or the CPC."

File: CHARTER.md [hunk 12]
```diff
@@ -0,0 +1,192 @@
...
+
+Subject to such policies as may be set by the CPC, the TSC voting members are responsible for all technical development 
+within the webpack project, including:
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Project governance and process.
+* GitHub repository hosting.
+* Conduct guidelines.
```
Comment (tobie) [inline CHARTER.md]: This isn't super clear. The CPC is responsible for the code of conduct. The TSC might have opted in to handle CoC reports. If it's the latter you'll want to clarify this.

File: CHARTER.md [hunk 13]
```diff
@@ -0,0 +1,192 @@
...
+* Maintaining the list of additional Collaborators.
```
Comment (tobie) [inline CHARTER.md]: Collaborators aren't defined as a term. Either define it or lowercase it. (note you can lowercase even if you define it, that makes the text easier to read and more approachable.)

File: CHARTER.md [hunk 14]
```diff
@@ -0,0 +1,192 @@
...
+Leadership roles in the webpack project will be peer elected representatives of the community.
+
+For election of persons (such as the TSC Chairperson), a multiple-candidate method should be used, such as:
+
+* [Condorcet][] or
+* [Single Transferable Vote][]
+
+Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one 
+position to be filled. No election is required if there is only one candidate and no objections to the candidate's 
+election. Elections shall be done within the projects by the Collaborators active in the project.
```
Comment (tobie) [inline CHARTER.md]: It looks like you're mixing up seats and candidates here. I suggest simplifying this section and copying it from what the CPC does for elections.

Comment (tobie) [inline CHARTER.md]: "Elections shall be done within the projects by the Collaborators active in the project." Totally unclear why would webpack projects elect people and what their roles would be. I suggest entirely removing this last sentence and moving a webpack project level governance outside of the charter.

File: CHARTER.md [hunk 15]
```diff
@@ -0,0 +1,192 @@
...
+
+The TSC voting members will elect from amongst voting TSC members a TSC Chairperson to work on building an agenda for
```
Comment (tobie) [inline CHARTER.md]: Don't you need two reps?

File: CHARTER.md [hunk 16]
```diff
@@ -0,0 +1,192 @@
...
+position to be filled. No election is required if there is only one candidate and no objections to the candidate's 
+election. Elections shall be done within the projects by the Collaborators active in the project.
+
+The TSC voting members will elect from amongst voting TSC members a TSC Chairperson to work on building an agenda for 
+TSC meetings and a OpenJS Cross Project Council (CPC) voting member to represent the TSC in the OpenJS Foundation for a 
+term of one year. The Chair and voting CPC member may be (but are not required to be) the same person. The TSC shall 
+hold annual elections to select a TSC Chairperson and voting CPC member; there are no limits on the number of terms a 
+TSC Chairperson or voting CPC member may serve.
+
+#### Section 4.2.2. Voting
```
Comment (tobie) [inline CHARTER.md]: Call this section decision-making

File: CHARTER.md [hunk 17]
```diff
@@ -0,0 +1,192 @@
...
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The TSC voting members shall establish 
+appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the 
+development process.
+
+The TSC voting members follow a [Consensus Seeking][] decision making model. When an agenda item has appeared to reach a
```
Comment (tobie) [inline CHARTER.md]: is "lazy consensus" and "Consensus Seeking decision making model" the same or different?

If it's the same, use the same words. If it is different define both and explain how they differ.

File: CHARTER.md [hunk 18]
```diff
@@ -0,0 +1,192 @@
...
+
+The TSC voting members will elect from amongst voting TSC members a TSC Chairperson to work on building an agenda for 
+TSC meetings and a OpenJS Cross Project Council (CPC) voting member to represent the TSC in the OpenJS Foundation for a 
+term of one year. The Chair and voting CPC member may be (but are not required to be) the same person. The TSC shall 
+hold annual elections to select a TSC Chairperson and voting CPC member; there are no limits on the number of terms a 
+TSC Chairperson or voting CPC member may serve.
+
+#### Section 4.2.2. Voting
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The TSC voting members shall establish
```
Comment (tobie) [inline CHARTER.md]: Define "internal projects" somewhere or better yet, consistently call them "webpack projects."

File: CHARTER.md [hunk 19]
```diff
@@ -0,0 +1,192 @@
...
+appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the 
+development process.
+
+The TSC voting members follow a [Consensus Seeking][] decision making model. When an agenda item has appeared to reach a 
+consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+For all votes, a simple majority of all TSC voting members for, or against, the issue wins. A TSC voting member may 
+choose to participate in any vote through abstention.
+
+All changes to this charter must be approved by the CPC.
```
Comment (tobie) [inline CHARTER.md]: What is this doing in the section on voting?

File: CHARTER.md [hunk 20]
```diff
@@ -0,0 +1,192 @@
...
+#### Section 4.2.2. Voting
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The TSC voting members shall establish 
+appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the 
+development process.
+
+The TSC voting members follow a [Consensus Seeking][] decision making model. When an agenda item has appeared to reach a 
+consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+For all votes, a simple majority of all TSC voting members for, or against, the issue wins. A TSC voting member may
```
Comment (tobie) [inline CHARTER.md]: When do you vote on things? you want to spell this out.

File: CHARTER.md [hunk 21]
```diff
@@ -0,0 +1,192 @@
...
+
+The TSC voting members follow a [Consensus Seeking][] decision making model. When an agenda item has appeared to reach a 
+consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+For all votes, a simple majority of all TSC voting members for, or against, the issue wins. A TSC voting member may 
+choose to participate in any vote through abstention.
+
+All changes to this charter must be approved by the CPC.
+
+#### Section 4.2.3. Project Roles
```
Comment (tobie) [inline CHARTER.md]:
```suggestion
#### Section 4.3. Other Project Roles
```

File: CHARTER.md [hunk 22]
```diff
@@ -0,0 +1,192 @@
...
+
+Individuals making significant and valuable contributions are made Collaborators and given commit-access to the project.
+These individuals are identified by the TSC and their addition as Collaborators is discussed during a TSC meeting. 
+Modifications of the contents of the git repository are made on a collaborative basis as defined in the development 
+process.
+
+Collaborators may opt to elevate significant or controversial modifications, or modifications that have not found 
+consensus to the TSC for discussion by assigning the `tsc-agenda` tag to a pull request or issue. The TSC voting members 
+should serve as the final arbiter where required. The TSC will maintain and publish a list of current Collaborators, as 
+well as a development process guide for Collaborators and Contributors looking to participate in the development effort.
```
Comment (tobie) [inline CHARTER.md]: Don't put implementation details in the charter
```suggestion
Collaborators may opt to elevate significant or controversial modifications, or modifications that have not found 
consensus to the TSC for discussion. The TSC voting members 
should serve as the final arbiter where required. The TSC will maintain and publish a list of current Collaborators, as 
well as a development process guide for Collaborators and Contributors looking to participate in the development effort.
```

File: CHARTER.md [hunk 23]
```diff
@@ -0,0 +1,192 @@
...
+consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+For all votes, a simple majority of all TSC voting members for, or against, the issue wins. A TSC voting member may 
+choose to participate in any vote through abstention.
+
+All changes to this charter must be approved by the CPC.
+
+#### Section 4.2.3. Project Roles
+
+The webpack git repository is maintained by the TSC and additional Collaborators who are added by the TSC voting members
```
Comment (tobie) [inline CHARTER.md]: What about other subprojects? This feels like an implementation detail that doesn't belong in the charter

File: CHARTER.md [hunk 24]
```diff
@@ -0,0 +1,192 @@
...
+Collaborators may opt to elevate significant or controversial modifications, or modifications that have not found 
+consensus to the TSC for discussion by assigning the `tsc-agenda` tag to a pull request or issue. The TSC voting members 
+should serve as the final arbiter where required. The TSC will maintain and publish a list of current Collaborators, as 
+well as a development process guide for Collaborators and Contributors looking to participate in the development effort.
+
+## Section 5. Definitions
+
+* **Contributors**: contribute code or other artifacts, but do not have the right to commit to the code base. 
+  Contributors work with the project's Collaborators to have code committed to the code base. A Contributor may be 
+  promoted to a Collaborator by the TSC. Contributors should rarely be encumbered by the TSC and never by the CPC or
```
Comment (tobie) [inline CHARTER.md]: encumbered feels like an unfortunate term, here. tbh.

File: CHARTER.md [hunk 25]
```diff
@@ -0,0 +1,192 @@
...
+well as a development process guide for Collaborators and Contributors looking to participate in the development effort.
+
+## Section 5. Definitions
+
+* **Contributors**: contribute code or other artifacts, but do not have the right to commit to the code base. 
+  Contributors work with the project's Collaborators to have code committed to the code base. A Contributor may be 
+  promoted to a Collaborator by the TSC. Contributors should rarely be encumbered by the TSC and never by the CPC or 
+  OpenJS Foundation Board.
+
+* **Project**: a technical collaboration effort, e.g. a subsystem, that is organized through the project creation
```
Comment (tobie) [inline CHARTER.md]: Use `webpack project` or `internal project` here to distinguish it from what the foundation calls projects.

File: CHARTER.md (hunk 22 above)
Comment (tobie) [inline CHARTER.md]: "development process guide" / "development process"; if it's the same doc, use consistent terminology.

File: CHARTER.md [hunk 26]
```diff
@@ -0,0 +1,192 @@
...
+
+## Section 5. Definitions
+
+* **Contributors**: contribute code or other artifacts, but do not have the right to commit to the code base. 
+  Contributors work with the project's Collaborators to have code committed to the code base. A Contributor may be 
+  promoted to a Collaborator by the TSC. Contributors should rarely be encumbered by the TSC and never by the CPC or 
+  OpenJS Foundation Board.
+
+* **Project**: a technical collaboration effort, e.g. a subsystem, that is organized through the project creation 
+  process and approved by the TSC.
```
Comment (tobie) [inline CHARTER.md]: what is the "project creation process" ?

**Review by tobie — COMMENTED** [review]
This is a huge step forward from the previous doc. Congrats, folks, great work!!

File: CHARTER.md (hunk 2 above)
Comment (ovflowd) [reply CHARTER.md]:
```suggestion
These repositories, along with others housed within the `github.com/webpack` and `github.com/webpack-contrib are owned by the webpack project.`.
```

Comment (ovflowd) [reply CHARTER.md]: Something like that?

File: CHARTER.md (hunk 3 above)
Comment (ovflowd) [reply CHARTER.md]: That's a good call, simplifying the list makes sense.

File: CHARTER.md (hunk 4 above)
Comment (ovflowd) [reply CHARTER.md]: @evenstensberg you wrote this entry, what did you have in mind with this? 🤔

File: CHARTER.md (hunk 12 above)
Comment (ovflowd) [reply CHARTER.md]: I believe this bullet list was a copy of the current Node.js's ones.

File: CHARTER.md (hunk 4 above)
Comment (evenstensberg) [reply CHARTER.md]: I meant external tools like CLIs tapping into webpack

**Comment by evenstensberg** [comment]
@tobie Thanks for the review. We will address the changes soon and re-request another one.

File: CHARTER.md (hunk 4 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed

File: CHARTER.md (hunk 6 above)
Comment (evenstensberg) [reply CHARTER.md]: we removed it, not critical.

File: CHARTER.md (hunk 24 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed

File: CHARTER.md (hunk 25 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 22 above)
Comment (evenstensberg) [reply CHARTER.md]: We use `development process` throughout the doc.

File: CHARTER.md (hunk 23 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed by removing the section.

File: CHARTER.md (hunk 7 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed by adding more specific terms.

File: CHARTER.md (hunk 2 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed

File: CHARTER.md (hunk 3 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 5 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 8 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 10 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 11 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 13 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 9 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 12 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 16 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 14 above)
Comment (evenstensberg) [reply CHARTER.md]: @ovflowd can you find the relevant text we need to change to?

File: CHARTER.md (hunk 26 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 22 above)
Comment (evenstensberg) [reply CHARTER.md]: We removed this section

File: CHARTER.md (hunk 21 above)
Comment (evenstensberg) [reply CHARTER.md]: We removed this

File: CHARTER.md (hunk 20 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 19 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 18 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 17 above)
Comment (evenstensberg) [reply CHARTER.md]: fixed.

File: CHARTER.md (hunk 14 above)
Comment (ovflowd) [reply CHARTER.md]: Sure!

Comment (ovflowd) [reply CHARTER.md]: Will leave them intentionally blank and let this be decided afterwards as part of the project governance outside of the charter.

**Comment by ovflowd** [comment]
I believe we're ready, @tobie!

File: CHARTER.md [hunk 27]
```diff
@@ -0,0 +1,137 @@
...
+voting membership shares an employer, then the situation must be immediately remedied by the removal of voting member
+status from one or more TSC voting members affiliated with the over-represented employer(s).
+
+The TSC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On
+Air, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson.
+Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC voting member.
+Minutes or an appropriate recording shall be taken and made available to the community through accessible public
+postings.
+
+TSC voting members are expected to regularly participate in TSC meetings and issue triaging. Meeting notes will be released such that absent members of the TSC can provide feedback in any way.
```
Comment (tobie) [inline CHARTER.md]:
```suggestion
TSC voting members are expected to regularly participate in TSC meetings and issue triaging.
```

Seems like the second part of that paragraph is a little redundant with the minute taking noted above.

File: CHARTER.md [hunk 28]
```diff
@@ -0,0 +1,137 @@
...
+appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance
+of a TSC meeting).
+
+Note that project operations remain subject to any relevant policy or process specified by the OpenJS Foundation board or Cross Project Council.
+
+### Section 4.2. Decision-making, Voting, and/or Elections
+
+#### Section 4.2.1. Elections
+
+These processes are described in webpack's [GOVERNANCE](GOVERNANCE.md) document.
```
Comment (tobie) [inline CHARTER.md:115]: It seems that the election process is actually described in section 3 above, no?

Maybe move it here?

File: CHARTER.md [hunk 29]
```diff
@@ -0,0 +1,137 @@
...
+#### Section 4.2.2. Decision Making
+
+For webpack projects' decisions, Collaborators and TSC voting members shall operate under Lazy Consensus ([consensus seeking][]). When an agenda item has appeared to reach a
+consensus, the moderator will ask "Does anyone object?" as a final call for dissent from the consensus. For all votes, a simple majority of all TSC voting members for, or against, the issue wins. A TSC voting member may
+choose to participate in any vote through abstention. Votes are needed to add new members, remove members from the TSC or deciding in project critical issues, such as roadmapping major versions of webpack core.
+
+
+## Section 5. Definitions
+
+* **Contributors**: contribute code or other artifacts, but do not have the right to commit to the code base.
```
Comment (tobie) [inline CHARTER.md]: The term isn't used anywhere in the charter. No point in defining it at all.

**Review by tobie — APPROVED** [review]
Some minor suggestions. Good to go otherwise. Congrats on all of the work done!

File: CHARTER.md (hunk 28 above)
Comment (ovflowd) [reply CHARTER.md:115]: I don't see anything related to elections on section 3 🤔

File: CHARTER.md (hunk 29 above)
Comment (ovflowd) [reply CHARTER.md]: I guess we can remove it in the opposite direction. Is there a term that would make sense to be added to the Definitions, in your opinion?

File: CHARTER.md (hunk 28 above)
Comment (tobie) [reply CHARTER.md:115]: Well, this is where you describe how TSC members are chosen, so that’s the closest you have to elections.

File: CHARTER.md (hunk 29 above)
Comment (tobie) [reply CHARTER.md]: Not that I think of.

File: CHARTER.md (hunk 28 above)
Comment (ovflowd) [reply CHARTER.md:115]: Let me see if I understood this correctly: you're asking for us to move this election sub-section to be part of section 3?

(The charter template on the CPC's repo mentions elections as part of section 4, that's why I added here) -- because there's no reference in section 3 of how someone becomes a TSC member.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by evenstensberg — COMMENTED** [review]
Could you let us know when we can merge this and begin bootstrapping the tsc?

File: CHARTER.md [hunk 30]
```diff
@@ -0,0 +1,132 @@
...
+- Supporting multiple output formats (e.g., AMD, CommonJS, ES Modules) for compatibility with various environments
+- Community-driven tools and best practices provided through
+  [plugins](https://webpack.js.org/plugins/) and [guides](https://webpack.js.org/guides/)
+- Technical help and discussions via community platforms such as
+  [GitHub Discussions](https://github.com/webpack/webpack/discussions)
+
+In addition to the main repository, `webpack` (bundler), the organization maintains and manages several other core
+projects that are integral to the ecosystem such as `webpack-dev-server`, `webpack-cli`,  and `webpack-contrib` projects.
+
+These repositories, along with others housed within the `github.com/webpack` and `github.com/webpack-contrib are owned by the webpack project.`.
```
Comment (bjohansebas) [inline CHARTER.md]: I'm not sure if this is on purpose
```suggestion
These repositories, along with others housed within the `github.com/webpack` and `github.com/webpack-contrib` are owned by the webpack project.
```

Comment (evenstensberg) [reply CHARTER.md]: When we wrote this, we wanted examples of which projects that is ours, but I think this is better. LGTM

**Review by bjohansebas — APPROVED** [review]
LGTM, even though I'm not a member of the CPC yet

**Review by joesepi — APPROVED** [review]
LGTM. Approving as a CPC member. ✌️ 

@bjohansebas fix is good to include though
